config/cloudinary.yml

```diff
@@ -0,0 +1,11 @@
+development:
+  enhance_image_tag: true
+  static_file_support: false
+
+test:
+  enhance_image_tag: true
+  static_file_support: false
+
+production:
```

Comment:
At first glance, this is what could possibly be inconsistent with what we currently do? I'm not sure, just an inkling.

I can dive deeper later.

Reply:
@benhalpern can you explain further what you mean here since it seems we want to get this pushed out